tools/clang/lib/SPIRV/PervertexInputVisitor.cpp

```diff
@@ -310,8 +317,28 @@ bool PervertexInputVisitor::visit(SpirvFunctionCall *inst) {
   unsigned argIndex = 0;
   for (auto *arg : inst->getArgs()) {
     if (currentFunc->getMappedFuncParam(arg)) {
-      createVertexStore(arg,
-                        createVertexLoad(currentFunc->getMappedFuncParam(arg)));
+      auto paramVar = currentFunc->getMappedFuncParam(arg);
```

Comment:
Could you move this outsize of the `if`, and use `paramVar` in the condition? This avoid two calls to the same function.

Reply:
Will fix.